### PR TITLE
SGT: React Feature 2 | View Average Grade - Kurt Yazdizadeh

### PR DESCRIPTION
**Pull request by kurtyazdizadeh**
Screen Shot 2020-04-18 at 12 52 12 PM

### CODE REVIEW

File: server/public/styles.css [hunk 1]
```diff
@@ -1,3 +1,6 @@
-h1 {
-  margin: 1rem 0;
+header {
+  margin: .75rem 0;
+}
+h4 {
+  text-align: right;
```
Comment (thebearingedge) [inline server/public/styles.css]: I believe there are helper classes for stuff like this in Bootstrap

https://getbootstrap.com/docs/4.4/utilities/spacing/

https://getbootstrap.com/docs/4.4/utilities/text/

File: client/components/app.jsx [hunk 2]
```diff
@@ -6,7 +6,8 @@ class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      grades: []
+      grades: [],
+      averageGrade: 0
```
Comment (thebearingedge) [inline client/components/app.jsx]: This isn't being used, you can remove it.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: server/public/styles.css (hunk 1 above)
Comment (kurtyazdizadeh) [reply server/public/styles.css]: I found the classes and used those instead

File: client/components/app.jsx (hunk 2 above)
Comment (kurtyazdizadeh) [reply client/components/app.jsx]: Forgot to remove it when I changed how to send the prop over to the Header component, fixed now

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**